src/mainviews/list-jobs.tsx

```diff
@@ -182,8 +186,8 @@ export function NotebookJobsListBody(
       name: trans.__('Output files')
     },
     {
-      sortField: 'start_time',
-      name: trans.__('Start time')
+      sortField: 'create_time',
+      name: trans.__('Create time')
```

Comment:
"Created" or "Created at" might be a more understandable column header for this.